infrastructure/templates/template.json

```
@@ -535,6 +535,29 @@
       "type": "bool",
       "defaultValue": false
     },
+    "openIdConnectSpaClientAllowOfflineAccess": {
+      "type": "bool",
+      "defaultValue": false,
+      "metadata": {
+        "description": "Whether or not this environment's Admin API supports refresh tokens"
+      }
+    },
+    "openIdConnectSpaClientRefreshTokenUsage": {
+      "type": "string",
+      "defaultValue": false,
+      "allowedValues": ["ReUse", "OneTimeOnly"],
+      "metadata": {
+        "description": "Whether or not this environment's Admin API supports reusable or one-time refresh tokens"
+      }
+    },
+    "openIdConnectSpaClientRefreshTokenExpiration": {
+      "type": "string",
+      "defaultValue": false,
```

Should the default value here be `"Absolute"` rather than `false`?

```suggestion
      "defaultValue": "Absolute",
```

---

Amended this one too